engine/consensus/matching/engine.go

```diff
@@ -501,19 +510,24 @@ func (e *Engine) checkingSealing() error {
 	sealableResultsSpan := e.tracer.StartSpanFromParent(sealingSpan, trace.CONMatchCheckSealingSealableResults)
 
 	// get all results that have collected enough approvals on a per-chunk basis
-	sealableResults, err := e.sealableResults()
+	sealableResults, nextUnsealeds, err := e.sealableResults()
 	if err != nil {
 		return fmt.Errorf("could not get sealable execution results: %w", err)
 	}
+
+	lg := e.log.With().
+		Int("sealable_results_count", len(sealableResults)).
+		Str("next_unsealed_results", nextUnsealedsPrintable(nextUnsealeds)).
+		Logger()
+
 	if len(sealableResults) == 0 {
-		// skip if no results can be sealed yet
+		lg.Info().Msg("checking sealing finished. No results can be sealed yet")
```

Comment:
When sealing is halt, this log would be very useful to confirm that whether it's because there is no incorporated result (which might be a bug) or some other cases. This helps narrow down the debugging scope.

In terms of volume, this log will be printed at most once every 2 seconds, which is fine. We would either print this log or the "Check sealing successful..." log. In a happy case, we are already printing logs every 2 second, so we are not increasing here. In an unhappy case, that we did print this, then we will skip printing the "Check sealing successful", so we are not increasing log volume either.